aten/src/ATen/core/dispatch/DispatchKeyExtractor.h

```diff
@@ -163,12 +163,12 @@ struct TORCH_API DispatchKeyExtractor final {
   }
 
   // NB: If there is no valid dispatch key, this will return Undefined
-  DispatchKey dispatchKeySetToDispatchKey_(
+  DispatchKeySet computeDispatchKeySetExcludingFallthroughKeys(
```

Comment:
(following up on previous thread) actually, I should've suggested inlining this into its callers - in the current codebase there are only two. So i.e. line 137 becomes
```
return impl::computeDispatchKeySet(ks, nonFallthroughKeys_);
```
and line 143 becomes
```
return impl::computeDispatchKeySet(ks, nonFallthroughKeys_ & eligibleKeys);
```

Reply:
totally agreed- I think my brain was looking at it backwards when I tried the refactor last week.